tests/test_job_management.py

```diff
@@ -2409,6 +2409,83 @@ def test_email_t1_t2(dispatcher_long_living_fixture,
         assert jdata["error_message"] == error_message
 
 
+@pytest.mark.parametrize("number_folders_to_delete", [1, 8])
+@pytest.mark.parametrize("soft_minimum_age_days", ["not_provided", 1, 5])
```

Comment:
you do not test hard limit?

Reply:
It's actually tested afterwards, simply because I thought of having it only as part of the configuration of the dispatcher (default to 30 days). Shall we have it as an argument for the endpoint?